velox/docs/monthly-updates/december-2023.rst

```diff
@@ -0,0 +1,99 @@
+
+Documentation
+=============
+
+* Add documentation for :doc:`Runtime Metric</metrics>`.
+
+Core Library
+============
+
+* Add support for spilling during the output processing stage of the ``OrderBy`` operator.
+* Add support for ``k range frames`` in ``Window`` operator.
+* Add support for aggregations over sorted inputs to ``StreamingAggregation``.
+* Add support for TTL in AsyncDataCache and SsdCache. :pr:`6412`
+* Add support for TypeSignature Parser using Flex and Bison. This is used to parse function signatures.
+* Add support for metrics related to memory arbitration and spilling. :pr:`7940`, :pr:`8025`
+* Add ``Status`` class that can be used to carry the success or error state of an operation.
+  This is similar to `arrow::Status <https://arrow.apache.org/docs/cpp/api/support.html#_CPPv4N5arrow6StatusE>`_.
+* Add support for ``cast(double|real as varchar)`` to return scientific notation when magnitude
+  of the input value is greater than or equal to 10^7, or less than 10^-3.
+* Add config ``max_split_preload_per_driver`` and remove the ``split_preload_per_driver`` flag.
+* Add config ``max_arbitrary_buffer_size`` to set the maximum size in bytes for a task's buffered
+  output when the output is distributed randomly among consumers. The producer drivers are blocked
+  when the buffer size exceeds this config.
+* Add config ``max_spill_bytes`` to bound the storage used for spilling. The default value is set to 100GB.
+  If it is set to zero, then there is no limit.
+* Fix reclaiming memory from hash build operators in grouped execution mode. :pr:`8178`
+* Fix non-termination of hash join in certain conditions. :pr:`7925`, :pr:`8012`.
+* Fix non-termination of the distinct aggregation in certain conditions. :pr:`7968`.
+* Fix ``LimitNode`` offset and count values from overflowing.
+* Fix accuracy of the DECIMAL type average computation. :pr:`7944`
+
+Presto Functions
+================
+
+* Add support for TIMESTAMP WITH TIME ZONE input type to :func:`format_datetime` function.
+* Add support for UNKNOWN key type to :func:`map_keys` and :func:`map_values` functions.
+* Add support for DECIMAL types to :func:`approx_distinct` aggregate function.
+* Fix :func:`find_first` to return NULL when the input ArrayVector is NULL but
+  has non-zero offsets and sizes.
+* Fix :func:`find_first` to support input ArrayVectors that are only NULL or empty.
+* Fix :func:`find_first` to return NULL for inputs NULL array and 0 index.
+* Fix :func:`find_first` to throw an error for inputs empty array and invalid start index.
+* Fix :func:`array_sort` to fail gracefully if the specified comparator lambda
+  is not supported.
+* Fix :func:`transform_keys` to check new keys for NULLs.
+* Fix :func:`set_union`, :func:`set_agg` to preserve the order of inputs.
+* Fix :func:`map` to produce the correct output if input arrays have NULL rows but with
+  invalid offsets and sizes.
+
+Spark Functions
+===============
+
+* Add :ref:`Expand <ExpandNode>` operator.
```

Comment:
Move this to Core Library section since this is not a function and not an expression or special form.